distributions/debian/control

```diff
@@ -17,7 +17,7 @@ Architecture: any
 # Define dependencies explicitly for the best compatibility across all supported Ubuntu/Debian versions.
 # The automatism would otherwise select package names or versions which are not available on some systems,
 # especially when run in the Github Ubuntu 18.04 build environment.
-Depends: libc6 (>= 2.17), libgcc1 (>= 1:3.0), libjack-jackd2-0 (>= 1.9.5~dfsg-14) | libjack-0.116, libqt5core5a (>= 5.5.0), libqt5gui5 (>= 5.0.2) | libqt5gui5-gles (>= 5.0.2), libqt5network5 (>= 5.0.2), libqt5widgets5 (>= 5.2.0), libqt5xml5 (>= 5.0.2), libstdc++6 (>= 5.2)
+Depends: libc6 (>= 2.17), libstdc++6 (>= 5.2), libgcc1 (>= 1:3.0), libjack-jackd2-0 (>= 1.9.5~dfsg-14) | libjack-0.116, libqt5core5a (>= 5.5.0), libqt5network5 (>= 5.0.2), libqt5xml5 (>= 5.0.2), libqt5gui5 (>= 5.0.2) | libqt5gui5-gles (>= 5.0.2), libqt5widgets5 (>= 5.2.0), libqt5multimedia5 (>= 5.15.2)
```

Comment:
Also moved `libstdc++6 (>= 5.2)` from last in line to just after `libc6` and moved all the non-headless stuff after the core dependenclies.  The order shouldn't actually matter.  So long as my cut-n-pastes worked...